app/src/main/java/com/example/weatherapp/domain/models/Models.kt

```diff
@@ -1,97 +1,64 @@
 package com.example.weatherapp.domain.models
 
-import com.google.gson.annotations.SerializedName
-
-data class ForecastWeather(var city: City, val list: List<ListElement>)
+data class ForecastWeather(var city: City, val list: List<ForecastItem>)
 
 data class CurrentWeather(
-    @SerializedName("name") var cityName: String,
-    val main: Main,
-    val coord: Coord,
+    var cityName: String,
```

Comment:
Make it val. It's a good style make data classes immutable. You can make copy when you need changes

Reply:
Done